requestUser problem in production

HirotoShioi opened the pull request:
## Description
meteor method requestUser is showing error that user does not exist in production.
<img width="1175" alt="2017-02-07 10 51 06" src="https://cloud.githubusercontent.com/assets/15665039/22674565/f8526c40-ed23-11e6-87a0-a5049ecb5960.png">

Commit:
g149 - Fix requestUser error

Description
Fix requestUser error occuring in the production by doing the user
validation only on server.
The problem was that Meteor users can be queried only on server side
and the publishment was not being reactive to the labelShare.

Solution
Fix the meteor method so that query will occur only on server side.
Use the publishComposite so that user collections will be reactive
to the labelShare.

zikaa commented:
Problem1: User collection on client side does not have all the user collection. Because of this, query on the client side was failing.

Problem2: User collection publish was not being reactive to the labelShare collection. Because of this, user collection was not updating reactively.

Commit:
g149 - Remove console.log

Description
Some console.logs that were used to investigate about the bug.

Solution
Remove console.log from template.
